pipeline_games/load_games.py

```diff
@@ -0,0 +1,256 @@
+"""Script for loading to database"""
+from os import environ
+from dotenv import load_dotenv
+import pandas as pd
+from psycopg2 import connect, DatabaseError, sql
+from psycopg2.extras import RealDictCursor, execute_batch
+
+
+def get_db_connection(config):
+    """Connect to the database with game data"""
+    try:
+        return connect(
+            user=config['DATABASE_USERNAME'],
+            password=config['DATABASE_PASSWORD'],
+            host=config['DATABASE_IP'],
+            port=config['DATABASE_PORT'],
+            database=config['DATABASE_NAME'])
+    except ValueError:
+        return "Error connecting to database."
+
+
+def execute_batch_columns(conn, data: pd.DataFrame, table: str, column: str, page_size=100):
+    """Using psycopg2.extras.execute_batch() to insert the dataframe"""
+    tuples = tuples = list(zip(data.unique()))
```

Comment:
I assume this is correct but `tuples = tuples = list(zip(data.unique()))` looks very strange. 

No feedback, just interesting to see that if thats how it is intended to be!

Reply:
omgosh no -- that is just meant to be one T T -- I will change it!